package.json

```diff
@@ -8,18 +8,15 @@
     "contracts:compile:abi": "typechain --target ethers-v5 --out-dir src/abis/types \"./src/abis/**/*.json\"",
     "contracts:compile:v3": "typechain --target ethers-v5 --out-dir src/types/v3 \"./node_modules/@uniswap/**/artifacts/contracts/**/*[!dbg].json\"",
     "contracts:compile": "yarn contracts:compile:abi && yarn contracts:compile:v3",
-    "relay": "relay-compiler relay.config.js",
-    "relay-thegraph": "relay-compiler relay_thegraph.config.js",
     "graphql:fetch": "node fetch-schema.js",
-    "graphql:generate": "yarn relay && yarn relay-thegraph",
     "apollo:generate:data": "graphql-codegen --config apollo-codegen.ts",
```

Comment:
These are really graphql-centric, not apollo-centric.
I'd prefer if these scripts could be renamed (to `graphql:generate`, `graphql:generate:data`, `graphql:generate:thegraph`), and the config files could be renamed from `apollo[,_thegraph].config.js` to `graphql[,_thegraph].config.js`.